dll/win32/shimgvw/shimgvw.c

```diff
@@ -921,9 +939,9 @@ ImageView_InitControls(HWND hwnd)
 
     if (shiSettings.Maximized) ShowWindow(hwnd, SW_MAXIMIZE);
 
-    hDispWnd = CreateWindowEx(WS_EX_CLIENTEDGE, WC_STATIC, _T(""),
-                              WS_CHILD | WS_VISIBLE,
-                              0, 0, 0, 0, hwnd, NULL, hInstance, NULL);
+    hDispWnd = CreateWindowExW(WS_EX_CLIENTEDGE, WC_STATIC, _T(""),
```

Comment:
You're explicitly using the Unicode variant of `CreateWindow(Ex)` yet in the `lpWindowName` parameter you're agnostic about which type of character string to use.
```suggestion
    hDispWnd = CreateWindowExW(WS_EX_CLIENTEDGE, WC_STATIC, L"",
```

Reply:
Argh, you are right. Fixed.